problem_builder/mixins.py

```diff
@@ -15,6 +15,20 @@
 
 loader = ResourceLoader(__name__)
 
+PB_LANGUAGE_JS_DIRECTORY_MAP = {
```

Comment:
@xitij2000 @wasifarbisoft Shouldn't this data be loaded from a configuration file instead of being hardcoded?